Documentation/PackageDescription.md

```diff
@@ -337,7 +341,7 @@ The dependency requirement can be defined as one of three different version requ
     minor version component if it adds new functionality in
     a backwards-compatible manner. And it should increment the patch
     version if it makes backwards-compatible bugfixes. To learn more about
-    the syntax of semantic versioning syntax, see `Version` or visit
+    semantic versioning syntax, see `Version` or visit
```

Comment:
Can we use a link instead of code formatting, like `[version](#version)`?

Reply:
Good catch. This has been updated.